docs/examples/frameworks/jax/pax-basic_example.ipynb

```diff
@@ -0,0 +1,1482 @@
+{
```

Comment:
<p>Nitpick:</p><p>This simple example shows how to train a neural network implemented in JAX with DALI <strong>data preprocessing</strong> <s>pipelines</s><em> (the pipelines can stay if you think it's clearer this way)</em></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>